modules/gapi/test/gapi_array_tests.cpp

```diff
@@ -28,6 +28,10 @@ G_TYPED_KERNEL(CountCorners,   <GScalar(GPointArray)>,  "test.array.in")
 {
     static GScalarDesc outMeta(const GArrayDesc &) { return empty_scalar_desc(); }
 };
+G_TYPED_KERNEL(PointIncrement, <GPointArray(GMat, GPointArray)>, "test.array.initialization")
```

Comment:
May be better to define operation ID as "point_increment" ?

Reply:
Fixed